### DISCUSSION TITLE
Directory tabbing autocompletion using the :ObsidianRename command?

### DISCUSSION THREAD

**Discussion by Enlightened-Meathead** [epwalsh/obsidian.nvim#560] [Q&A]
Just wanted to say I love the plugin and appreciate all your hard work! I like to have my notes in nested directories, so when i create a new note just starting in the root directory of my vault, if I want to rename a note and put it into a directory, when using the :ObsidianRename command I have to type the full path like Cyber/Pentest/Kali/Tools/the_note_name. This is a bit cumbersome as tabbing auto-completion doesn't work and when pressing tab it just appends ^I to the string for the new ID. Is there a setting/config I am missing to make this possible? Or, is this a feature that has not been added when using the rename command? When using just :w Cyber/.../the_note_name, my neovim does autocomplete with tabs. I assume when using the rename command it isn't just using the plain vim command line and some sort of Obsidian environment?

**Comment by epwalsh** [comment]
Hey @Enlightened-Meathead, I just added that in 79f7f2a.

**Reply by Enlightened-Meathead** [reply]
Thank you very much! Keep up the great work :)
